week-2/mandatory/3-api/server.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express");
+const app = express();
+const { Pool } = require("pg");
+
+const db = new Pool({
+  user: "44788",
+  host: "localhost",
+  database: "cyf_ecommerce",
+  password: "*********",
+  port: 5432,
+});
+
+app.get("/", (req, res) => {
+  res.send("Hello Wellcome to CYF ecommerce App ");
+});
+
+app.get("/customers", function (req, res) {
+  db.query(`SELECT * FROM customers`, (error, result) => {
+    if (error) throw error;
+    res.json(result.rows);
+  });
+});
```

Comment:
you're throwing the error but better to catch it somewhere (then and catch)

Reply:
sending the user a better message (data unavailable ie)

Reply:
working on catching errors